pkg/processor/timestamp/go.mod

```diff
@@ -0,0 +1,42 @@
+module github.com/signalfx/splunk-otel-collector/processor/timestampprocessor
+
+go 1.18
```

Comment:
the project is on 1.19 so it's probably best to match that. Being in step could be problematic for custom builds depending on the use case though...

Reply:
Actually why do we need this to be added as a module? I don't think we have other components as modules in our distro

Reply:
We have other asks to move to modules. We are preparing for this.

Reply:
Makes sense